bindings/java/src/main/native/Makefile.am

```diff
@@ -50,6 +50,10 @@ $(JUCX_GENERATED_H_FILES): $(STAMP_FILE)
 
 # Return CPU architecure in Java format
 arch = $(if $(filter x86_64,$(shell uname -p)),amd64,$(shell uname -p))
+
+# Append non-x86_64 arch as filename suffix
+suffix = $(if $(filter-out x86_64,$(shell uname -p)),-$(shell uname -p))
```

Comment:
don't we have multiple archs in same jar?

Reply:
ARM JARs have the specific platform name appended to their filenames, while x86_64 JARs retain their original filenames. This decision was made after a discussion in an email thread about separating JARs by platform.